packages/communication-react/review/communication-react.api.md

```diff
@@ -1342,6 +1342,7 @@ export interface VideoGalleryProps {
     onRenderLocalVideoTile?: (localParticipant: VideoGalleryLocalParticipant) => JSX.Element;
     onRenderRemoteVideoTile?: (remoteParticipant: VideoGalleryRemoteParticipant) => JSX.Element;
     remoteParticipants?: VideoGalleryRemoteParticipant[];
+    remoteVideoStreamLimitation?: number;
```

Comment:
Open question, should this be exposed to the composite interface and set by the sample itself?

Reply:
Actually, lets say no and be customer driven. Never mind.